web/components/Home/Resin/Subtractor.tsx

```diff
@@ -41,7 +42,7 @@ const Subtractor = ({ current }: { current: number }) => {
 
 const SubtractButton = ({ value, onClick }: { value: number; onClick: () => void }) => {
   const { formatMessage } = useIntl();
-  const hotkey = value.toString().slice(0, 1);
+  const hotkey = Math.abs(value).toString().slice(0, 1);
```

Comment:
> Or maybe use a combination hotkey for additions, like `shift+number`?

I think this is a good approach since it's backwards compatible, while also allowing additions with keyboard shortcuts